README.md

```diff
@@ -26,28 +26,33 @@ This value can be overriden using `SCR_LOGGING_FORMAT` environment variable.
 Alternatively, an external `logback.xml` with much more customizations can be provided using `-Dlogback.configurationFile` JVM parameter.
```

Comment:
Need to add logback and update this part of readme

Reply:
Can just copy what was used in SCR or 111?